dashboard/src/modules/components/OverviewComponent/SavedRunsComponent.jsx

```diff
@@ -0,0 +1,186 @@
+import "./index.less";
+
+import {
+  ActionsColumn,
+  InnerScrollContainer,
+  OuterScrollContainer,
+  TableComposable,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from "@patternfly/react-table";
+import { findNoOfDays, formatDateTime } from "utils/dateFunctions";
+import {
+  publishDataset,
+  setSelectedRuns,
+  updateDataset,
+} from "actions/overviewActions";
+import { useDispatch, useSelector } from "react-redux";
+
+import { ProgressBar } from "./common-component";
+import React from "react";
+
+const SavedRunsComponent = () => {
+  const dispatch = useDispatch();
+  const { savedRuns, selectedRuns } = useSelector((state) => state.overview);
+  const loginDetails = useSelector((state) => state.userAuth.loginDetails);
+
+  /* Selecting */
+  const areAllRunsSelected =
+    savedRuns?.length > 0 && savedRuns?.length === selectedRuns?.length;
+  const selectAllRuns = (isSelecting) => {
+    dispatch(setSelectedRuns(isSelecting ? savedRuns.map((r) => r) : []));
+  };
+  const onSelectRuns = (run, __rowIndex, isSelecting) => {
+    const otherSelectedRuns = selectedRuns.filter(
+      (r) => r.resource_id !== run.resource_id
+    );
+    const c = isSelecting ? [...otherSelectedRuns, run] : otherSelectedRuns;
+    dispatch(setSelectedRuns(c));
+  };
+  const isRowSelected = (run) =>
+    selectedRuns.filter((item) => item.name === run.name).length > 0;
+  /* Selecting */
+
+  /* Actions Row */
+  const moreActionItems = (dataset) => [
+    {
+      title:
+        dataset.metadata["dataset.access"] === "public"
+          ? "Publish"
+          : "Unpublish",
+      onClick: () => {
+        const accessType =
+          dataset.metadata["dataset.access"] === "public"
+            ? "private"
+            : "public";
+        dispatch(publishDataset(dataset, accessType));
+      },
+    },
+    {
+      title: dataset.metadata["global.dashboard.seen"]
+        ? "Mark unread"
+        : "Mark read",
+      onClick: () =>
+        dispatch(
+          updateDataset(
+            dataset,
+            "read",
+            !dataset.metadata["global.dashboard.seen"]
+          )
+        ),
+    },
+    {
+      title: "Delete",
+      onClick: () => dispatch(deleteDataset(dataset)),
+    },
+  ];
+  /* Actions Row */
+  const makeFavorites = (dataset, isFavoriting = true) => {
+    dispatch(updateDataset(dataset, "favorite", isFavoriting));
+  };
+  const calculatePercent = (dateStr) => {
+    const daysLeft = findNoOfDays(dateStr);
+    return Math.abs(daysLeft / 1000) * 100;
+  };
```

Comment:
You don't need to call `.abs()` here -- `findNoOfDays()` will never return a negative value.

Reply:
As the progress bar is removed this function is not needed anymore